src/Enchantments.h

```diff
@@ -111,6 +111,9 @@ class cEnchantments
 
 	// tolua_end
 
+	/** Merge enchantments, check for valid enchantments on item type */
```

Comment:
You should document what the `a_ItemType` and `a_MergeItemType` are. If one is related to `a_Other`, it would be good to name them similarly.